src/config/entities/__tests__/configuration.ts

```diff
@@ -23,7 +23,7 @@ export default (): ReturnType<typeof configuration> => ({
   },
   application: {
     isProduction: faker.datatype.boolean(),
-    runMigrations: true,
+    runMigrations: false,
```

Comment:
I think this will have side effects on targeted messages. cc @hectorgomezv

Reply:
Done https://github.com/safe-global/safe-client-gateway/commit/2ea0e5d8a752fd31cff745ff3490d8fe8134ddc7